spec/requests/posts_spec.rb

```diff
@@ -3,8 +3,23 @@
 RSpec.describe 'Posts', type: :request do
   describe 'GET /index' do
     it 'returns http success' do
-      get '/posts/index'
+      get '/users/:user_id/posts'
       expect(response).to have_http_status(:success)
     end
+
+    it 'displays the intended template' do
+      get '/users/:user_id/posts'
+      expect(response).to render_template('index')
+    end
+
+    it 'returns the http sucess' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'displays the intended template' do
+      get '/users/:user_id/posts/:id'
+      expect(response).to render_template('show')
+    end
```

Comment:
- [ ] Kindly include tests for the `Post#show` action that represents If the response body includes correct placeholder text. as per this [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for,includes%20correct%20placeholder%20text.).